pallets/msa/src/tests/mock.rs

```diff
@@ -45,6 +45,11 @@ frame_support::construct_runtime!(
 
 // See https://paritytech.github.io/substrate/master/pallet_collective/index.html for
 // the descriptions of these configs.
+parameter_types! {
+	pub MaxProposalWeight: frame_support::weights::Weight =
+		Perbill::from_percent(50) * common_runtime::constants::MAXIMUM_BLOCK_WEIGHT;
```

Comment:
🤔

Reply:
It is the same value as Polkadot uses. I believe it to be ok, but we will want to test governance upgrade on testnet.

Reply:
This value is in the unit testing mocks, which needed to be changed to pass the unit tests.
The value defined in runtime is (`runtime/frequency/src/lib.rs`):
```rust
type MaxProposalWeight = MaxCollectivesProposalWeight;

// Which resolves to this:
pub MaxCollectivesProposalWeight: Weight = Perbill::from_percent(50) * RuntimeBlockWeights::get().max_block;
```